packages/cli/src/commands/screenshot-diff/screenshot-diff.command.ts

```diff
@@ -36,24 +36,23 @@ export const diffScreenshots: (options: {
   headReplayId: string;
   baseScreenshot: PNG;
   headScreenshot: PNG;
-  threshold: number | null | undefined;
-  pixelThreshold: number | null | undefined;
+  diffOptions: ScreenshotDiffOptions;
   exitOnMismatch: boolean;
 }) => Promise<void> = async ({
   client,
   baseReplayId,
   headReplayId,
   baseScreenshot,
   headScreenshot,
-  threshold: threshold_,
-  pixelThreshold,
+  diffOptions,
   exitOnMismatch,
 }) => {
+  const { diffThreshold, diffPixelThreshold } = diffOptions;
   const logger = log.getLogger(METICULOUS_LOGGER_NAME);
 
-  const threshold = threshold_ || DEFAULT_MISMATCH_THRESHOLD;
```

Comment:
Previously there was a bug here in that it was impossible to set the threshold to 0 (this is a danger of using `||` by default instead of `??`)